blog/service/core/src/main/kotlin/com/excref/kotblog/blog/service/category/CategoryService.kt

```diff
@@ -0,0 +1,35 @@
+package com.excref.kotblog.blog.service.category
+
+import com.excref.kotblog.blog.service.category.domain.Category
+
+/**
+ * @author Ruben Vardanyan
+ * @since 06/07/2017 12:28
+ */
+interface CategoryService {
+    /**
+     * Creates a new category with the given name
+     *
+     * @param   name The category name
+     * @return  Persistent category
+     * @throws  com.excref.kotblog.blog.service.tag.exception.TagAlreadyExistsForNameException If the category already exists for the given name
```

Comment:
change from `TagAlreadyExistsForNameException` to `CategoryAlreadyExistsForNameException`